pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.towncrier]
+# Read https://github.com/ethereum/lahja/newsfragments/README.md for instructions
+package = "lahja"
+filename = "docs/release_notes.rst"
```

Comment:
It seems the release notes are currently in file `release.rst` so this doesn't seem to point to the correct file. I personally would prefer to keep this as `docs/release_notes.rst` but rename the `release.rst` file accordingly and change the name in the index where it is included. Make sure to run `make docs` to try out if the final result works as expected.

Reply:
I see that you renamed the file to `release_notes` but I'm a little bit confused because I would expect that to at least trigger one more change in the table of contents. Did you make sure to run `make docs` to see if everything builds cleanly? I suspect that it should fail because (at least) the toc still points to `releases.rst`.